maintainers/maintainer-list.nix

```diff
@@ -3391,12 +3391,6 @@
     githubId = 2129135;
     name = "Frederik Rietdijk";
   };
-  friedelino = {
```

Comment:
Why are you reverting it here?

Reply:
argh! alright. because i in the beginning i did not have the editing of maintainers file in a seperate commit. i'll try to fix this...